src/mocks/handlers.js

```diff
@@ -132,7 +132,7 @@ Object.keys(groupedResponses).forEach((endpoint) => {
     if (typeof response === "number") {
       status = response;
       response = undefined;
-    } else if (/post|path|put/.test(method)) {
+    } else if (/post|path|put|delete/.test(method)) {
```

Comment:
The correct code in getMatchingParamString should be:

```js
  if (/post|patch|put/.test(method)) {
    try {
      const bodyParams = await request.json();
      Object.entries(bodyParams).forEach(([key, value]) =>
        requestParams.set(key, value));
    } catch {}
  }
  ```
  
Note that there were three problems: the patch misspelling, the missing try, and the wrong array deconstruction syntax.

After this fix, sending an empty body should work, and parameters in the body should be also be translated to query parameters for the responses. So instead of a dynamic handler you can use these two static responses for validating codes:

```ts
 // Request IVA verification with correct code
  "POST /api/auth/rpc/ivas/:iva_id/validate-code?verification_code=122456": 204,

  // Request IVA verification with others codes
  "POST /api/auth/rpc/ivas/:iva_id/validate-code": 401,
 ```
 
 Similar for verifying TOTP codes.
 
 Can you include these fixes and changes in this PR?

Reply:
I'm afraid the simplified response does not work even with the fix. The response handler always returns the 401 even if it's meant to do the 204 response because the parameter matches.

Reply:
Are you sure the URLs match? There was some confusion between /api/auth and /api/wps URLs.

Reply:
Yep, the URLs matched. I already fixed the WPS issue. The response is 401 regardless of code submitted when that response setup is coded in